packages/nodes-base/nodes/Redis/Redis.node.ts

```diff
@@ -631,6 +751,37 @@ export class Redis implements INodeType {
 								const clientPublish = util.promisify(client.publish).bind(client);
 								await clientPublish(channel, messageData);
 								returnItems.push(items[itemIndex]);
+							} else if (operation === 'push'){
+								const redisList = this.getNodeParameter('list', itemIndex) as string;
+								const messageData = this.getNodeParameter('messageData', itemIndex) as string;
```

Comment:
When pushing data to the list, the description says that I can provide comma-separated values to the list. However, it pushes to the list as a single value when I do that. Am I missing something? Because it looks to me that the description it's not accurate.

<img width="428" alt="image" src="https://user-images.githubusercontent.com/16496553/177325465-49249a66-398c-4972-823e-b6e0eda9c522.png">

<img width="811" alt="image" src="https://user-images.githubusercontent.com/16496553/177325424-0e24f9b4-e377-42fc-8a24-4bcb8d7db1fb.png">

Reply:
That hint shouldn't exist. Removed it, the push `messageData` is now equivalent to the publish `messageData`.